build.sbt

```diff
@@ -252,6 +263,7 @@ def testTask(task: TaskKey[Tests.Output]) =
 lazy val catsVersion   = "0.8.1"
```

Comment:
Maybe we can also bump the cats version to `0.9.0` on the same occasion? Since I am going to update all my libraries with those new effects, I might as well to cats.